match/greenwood_pd.py

```diff
@@ -1,37 +1,51 @@
-from datamatch import ThresholdMatcher, JaroWinklerSimilarity, NoopIndex
+from datamatch import ThresholdMatcher, JaroWinklerSimilarity, ColumnsIndex
 from lib.path import data_file_path, ensure_data_dir
 import pandas as pd
 import sys
 
 sys.path.append("../")
 
 
-def match(gw_cprr, post_pprr):
-    post_pprr = post_pprr.loc[post_pprr.agency == "greenwood pd"].set_index(
-        "uid", drop=False
+def match_cprr_and_post(cprr, post):
+    dfa = (
+        cprr.loc[cprr.uid.notna(), ["uid", "first_name", "last_name"]]
+        .drop_duplicates(subset=["uid"])
+        .set_index("uid", drop=True)
     )
-    dfa = gw_cprr[["first_name", "last_name", "mid"]].drop_duplicates().set_index("mid")
-    dfb = post_pprr[["last_name", "first_name"]].drop_duplicates()
+    dfa.loc[:, "fc"] = dfa.first_name.fillna("").map(lambda x: x[:1])
+
+    dfb = (
+        post[["uid", "first_name", "last_name"]]
```

Comment:
Need to filter only POST rows for Greenwood PD here.

Reply:
I think that this is addressed.